### PR TITLE
Fixing write error in material

### PR DESCRIPTION
**Pull request by saransh13**
check if tThWidth is present in planeData before trying to convert to degrees. This was leading to an error during state write.

### CODE REVIEW

File: hexrd/material.py [hunk 1]
```diff
@@ -731,7 +731,10 @@ def dump_material(self, file, path=None):
         AtomInfo['stiffness'] = self.unitcell.stiffness
         AtomInfo['hkls'] = self.planeData.getHKLs()
         AtomInfo['dmin'] = self.unitcell.dmin
-        AtomInfo['tThWidth'] = numpy.degrees(self.planeData.tThWidth)
+        if self.planeData.tThWidth is None:
+            AtomInfo['tThWidth'] = Material.DFLT_TTH
```
Comment (psavery) [inline hexrd/material.py:735]: Isn't `Material.DFLT_TTH` in radians? I think you need to convert it to degrees here.

Comment (joelvbernier) [reply hexrd/material.py:735]: @saransh13 -- have you verified?

**Comment by saransh13** [comment]
Yea, tested with GUI master.

File: hexrd/material.py (hunk 1 above)
Comment (psavery) [reply hexrd/material.py:735]: In the two lines below this, the plane data `tThWidth` is being converted to degrees. The `Material.DFLT_TTH` must be converted to degrees as well.

**Review by joelvbernier — APPROVED** [review]

**Pull request merged**